packages/boost-demo/src/App.tsx

```diff
@@ -193,3 +232,16 @@ function App() {
 }
 
 export default App;
+// Stolen from chatgpt. It generates a random 65 byte hex string
+function generateRandomSignature(): string {
+  const characters = "0123456789abcdef";
+  let result = "0x";
+
+  const length = 132;
+  for (let i = 0; i < length; i++) {
+    const randomIndex = Math.floor(Math.random() * characters.length);
+    result += characters.charAt(randomIndex);
+  }
+
+  return result;
+}
```

Comment:
Might be overkill for a "bad" signature? Could we just use all zeros, or hard code some random string?

Reply:
https://github.com/statechannels/nitro-gui/pull/88/commits/4d6121b1e7bf830c7e6b0bcf28cedf10200fd8a5